src/Fixer/Phpdoc/PhpdocNoAliasTagFixer.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\Phpdoc;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\ConfigurationException\InvalidFixerConfigurationException;
+use PhpCsFixer\DocBlock\DocBlock;
+use PhpCsFixer\Tokenizer\Tokens;
+
+/**
+ * Case sensitive tag replace fixer (does not process inline tags like {@inheritdoc}).
+ *
+ * @author Graham Campbell <graham@alt-three.com>
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ * @author SpacePossum
+ */
+final class PhpdocNoAliasTagFixer extends AbstractFixer
+{
+    /**
+     * @var array<string, string>
+     */
+    private $configuration;
+
+    private static $defaultConfiguration = array(
+        'property-read' => 'property',
+        'property-write' => 'property',
+        'type' => 'var',
+        'link' => 'see',
+    );
+
+    /**
+     * Key value pairs of string, replace from -> to tags (without '@').
+     *
+     * @param string[]|null $configuration
+     */
+    public function configure(array $configuration = null)
+    {
+        if (null === $configuration) {
+            $this->configuration = self::$defaultConfiguration;
+
+            return;
+        }
+
+        $this->configuration = array();
+        foreach ($configuration as $from => $to) {
+            if (!is_string($from)) {
+                throw new InvalidFixerConfigurationException($this->getName(), 'Tag to replace must be a string.');
+            }
+
+            if (!is_string($to)) {
+                throw new InvalidFixerConfigurationException($this->getName(), sprintf('Tag to replace to from "%s" must be a string.', $from));
+            }
+
+            if (1 !== preg_match('#^[^\s]+$#', $to) || false !== strpos($to, '*/')) {
```

Comment:
sorry, i don't get this line
especially part with phpdoc ending.

why not check is it matchin `[a-zA-Z0-9]` ? spaces are not allowed, but `foo*bar` is not allowed as well.